### ISSUE TITLE
[BUG] AV1 videos don't load thumbnails

### ISSUE THREAD

**Issue by hewiofhiowa** [anilbeesetti/nextplayer#1144]
Next player doesn't load thumbnails for videos in the av1 video codec. It plays the video just fine, but does not load the thumbnail.

**Comment by anilbeesetti** [comment]
Hey @hewiofhiowa, This is a known issue and will try to resolve this in the next couple of updates

**Comment by Pentaphon** [comment]
Yes this is very strange considering that this is a modern codec that Media3 should have no issues with.

**Comment by anilbeesetti** [comment]
Will work from version 0.18.0, if it doesn't work please reopen the issue
